package.json

```diff
@@ -217,6 +219,7 @@
     "isomorphic-fetch": "^2.2.1",
     "jest": "^23.6.0",
     "jest-image-snapshot": "^4.0.2",
+    "jsdoc": "^3.6.6",
```

Comment:
This is the simplest, most common doc generation tool and shouldn't have any wider impacts that teams can use it generate local documentation if they'd like.